src/features/CourtCaseList/CourtCaseListEntry/CaseDetailsRow/CaseDetailsRow.tsx

```diff
@@ -96,7 +110,11 @@ export const CaseDetailsRow = ({
           </ConditionalRender>
         </Table.Cell>
         <Table.Cell className={caseDetailsCellClass}>
-          <Link href={caseDetailsPath(errorId)} id={`Case details for ${defendantName}`}>
+          <Link
+            onClick={async (event) => lockAndOrViewCase(event)}
+            href="" // emply href enables expected mouseover behaviour
```

Comment:
I wonder if we can use `<ActionLink>` component here. It does generate a link-like button without the need to set the href attribute.

Reply:
:eyes:

Reply:
Have swapped it out. Basically the same implementation, but without the need for an empty href and preventing default behaviour.